Automattic/wp-calypso
# Happy blocks: Update header desktop and mobile

## Proposed Changes

This PR updates the `happy blocks` header according to [new designs](IceKRdz5qlPYwuqzu6pBgF-fi-2272_29779), It contains mainly mobile changes, a divider added intentionally below the menu, the search new location and menu colors that now are blue.


Before:
<img width="1728" alt="image" src="https://user-images.githubusercontent.com/2653810/235669083-5c720977-08d2-4f44-ba87-9b9d618ba553.png">

<img width="1727" alt="image" src="https://user-images.githubusercontent.com/2653810/235668805-57332bef-6b03-427c-9f56-030007daab83.png">

<img width="408" alt="image" src="https://user-images.githubusercontent.com/2653810/235668872-8ba1076b-36d3-4cda-bb1a-9cd5f8eb7af5.png">
<img width="408" alt="image" src="https://user-images.githubusercontent.com/2653810/235668921-96d30e88-6053-4c55-9995-29e413201d60.png">


After:

<img width="1726" alt="image" src="https://user-images.githubusercontent.com/2653810/235669206-1c3431e3-d4a5-48b9-b461-5c92f546fc95.png">

<img width="1727" alt="image" src="https://user-images.githubusercontent.com/2653810/235669263-af697474-3b03-4316-8f7b-7907e329fbf2.png">

<img width="406" alt="image" src="https://user-images.githubusercontent.com/2653810/235669333-0ae486f9-9914-4fa1-91bd-28c814ea6fff.png">

<img width="411" alt="image" src="https://user-images.githubusercontent.com/2653810/235669391-562f08cb-926c-4f56-a57c-16110408e8b5.png">




## Testing Instructions

- Pull and run this branch
- Navigate to `cd apps/happy-blocks` and run `yarn dev --sync`
- Sandbox `learncft.wordpress.com` and `wordpress.com/support` and visit the site
- Check if the designs are matching and there's no weirdness around the header. 
- Check [learn](https://learncft.wordpress.com/) and mainly [support](https://wordpress.com/support/) since it is live
- Check mobile and desktop.


I have checked RTL and looks fine but also good, a second eye.

@vinimotaa, your check here would be highly appreciated 🙇
